edd_utils/__init__.py

```diff
@@ -137,6 +137,71 @@ def export_study(session, slug, edd_server='edd.jbei.org', verbose=True):
     return df
 
 
+#Get metadata
+def export_metadata(session, slug, edd_server='edd.jbei.org', verbose=True):
```

Comment:
Perhaps rename to `export_line_metadata`, because you're only fetching the line metadata from a study.

Reply:
Now gets the full study